wherehows-web/app/templates/search.hbs

```diff
@@ -1,114 +1,119 @@
 <div class="row">
   <div class="col-md-3 wh-sidebar">
-    {{#search/filter-rail header=header as |rail|}}
 
-      {{#search/containers/search-sources currentSource=source as |searchSource|}}
-        {{rail.radioGroup
-          radioOptions=searchSource.radioSources
-          title="Source"
-          changed=(action "sourceDidChange")}}
-      {{/search/containers/search-sources}}
+    {{#search/containers/search-facets
+      counts=facetCounts
+      selections=(readonly facetsSelections)
+      onFacetsChange=(action onFacetsChange) as |searchFacet selections counts onFacetChange onFacetClear|}}
 
-      {{!--Api for date range not currently available--}}
-      {{!--rail.dateRange
-        startDateChanged=(action 'startDateDidChange')
-        endDateChanged=(action 'endDateDidChange')--}}
-    {{/search/filter-rail}}
+      {{#each searchFacet as |facet|}}
+        {{search/search-facet
+          facet=facet
+          selections=(get selections facet.name)
+          counts=(get counts facet.name)
+          onChange=(action onFacetChange)
+          onClear=(action onFacetClear)}}
+      {{/each}}
+    {{/search/containers/search-facets}}
   </div>
   <div class="col-md-9 wh-main-content">
-    <section class="search-results page-section-container">
-      {{#if model.count}}
-        <div class="search-pagination">
-          <ul class="pager" style="margin-top: 0">
-            {{#unless first}}
-              <li class="previous">
-                {{#link-to 'search'
-                           (query-params categroy=category source=source page=previousPage)}}
-                  &larr; Prev
-                {{/link-to}}
+    {{#if searchLoading}}
```

Comment:
As mentioned above, using ember-concurrency will give you this derived state for free